integration_tests/tests/integration/simulation/model.rs

```diff
@@ -0,0 +1,62 @@
+use janus_messages::{CollectionJobId, Duration, Interval, Time};
+
+#[derive(Debug, Clone)]
+pub(super) struct Input {
+    pub(super) is_fixed_size: bool,
```

Comment:
Can we eliminate this field? It appears to only be used for constructing a `janus_aggregator_core::task::QueryType`. Perhaps it can be a verbatim `pub(super) query_type: QueryType` either in `Input` or `Config`?

Reply:
(Or perhaps there's some subtlety with Arbitrary that I'm missing--if so then a comment justifying the greater purpose of this field is justified).